.github/workflows/package-build.yml

```diff
@@ -112,7 +112,6 @@ jobs:
             ./build/*.zip
             ./build/modules/test/moduletest
             ./build/modules/bin/*.so
-            ./src/tests/universal-nrp-e2e/*.Tests.ps1
```

Comment:
don't remove this, this is needed for the Universal NRP Tests

Reply:
It looks like it is not used there. The `universalnrp-test-run` workflow does a checkout before comying/running the tests: https://github.com/Azure/azure-osconfig/blob/457201988e19dfcf37b70793a26868be2faeb366/.github/workflows/universalnrp-test-run.yml#L35-L45

Reply:
If we need it in an artifact we should probably put it in a different one since it changes the artifact output contents (adds the build directory structure + breaks E2E tests).

Reply:
ahh yes, didn't see that. Actually, this would fail anyhow if it wouldn't find it.